docs/common/error-utilities.ts

```diff
@@ -270,6 +271,9 @@ const RENAMED_PAGES: Record<string, string> = {
   '/tutorial/sharing/': '/tutorial/introduction/',
   '/tutorial/text/': '/tutorial/introduction/',
 
+  // Redirects for removed /archived pages
+  '/archived/': '/',
```

Comment:
Wouldn't https://docs.expo.dev/archive/ be a better redirect than the home page?

Reply:
Yes! Sorry about that. I got mixed up in the archive and archived 🤦
Thanks for catching that!

Reply:
Fixed it!